### PR TITLE
Added sample ssh commands to connect to Control and Worker node

### PR DESCRIPTION
**Pull request by kiranchavala**

### CODE REVIEW

File: source/plugins/cloudstack-kubernetes-service.rst [hunk 1]
```diff
@@ -153,6 +153,20 @@ A new network offering named DefaultNetworkOfferingforKubernetesService has been
    - Multi-control nodes, HA cluster can be created for Kubernetes version 1.16 and above only.
    - While creating multi-control nodes, HA cluster over a shared network, an external load-balancer must be manually setup. This load-balancer should have port-forwarding rules for SSH, Kubernetes API server access. Service assumes SSH access to cluster nodes is available from port 2222 to (2222 + cluster node count -1). Similarly, for API access 6443 must be forwarded to control nodes. Over the CloudStack isolated network these rules are automatically provisioned.
 
+.. parsed-literal::
```
Comment (DaanHoogland) [inline source/plugins/cloudstack-kubernetes-service.rst:156]: should this line only apply to the actual ssh command lines?

**Comment by DaanHoogland** [comment]
@blueorangutan docbuild

File: source/plugins/cloudstack-kubernetes-service.rst [hunk 2]
```diff
@@ -153,6 +153,20 @@ A new network offering named DefaultNetworkOfferingforKubernetesService has been
...
+
+   Example to ssh into the Control node and worker node
+
+   Control node
+
+   ssh -i <ssh-private.key > -p 2222 cloud@<Public ip address of Virtual Router>
+   
+   Worker node
+   
+   ssh -i <ssh-private.key > -p 2223 cloud@<Public ip address of Virtual Router>
```
Comment (DaanHoogland) [inline source/plugins/cloudstack-kubernetes-service.rst:166]:
```suggestion

Examples to ssh into the Control node and worker node

Control node

.. parsed-literal::

   ssh -i <ssh-private.key > -p 2222 cloud@<Public ip address of Virtual Router>
   
Worker node
   
.. parsed-literal::

   ssh -i <ssh-private.key > -p 2223 cloud@<Public ip address of Virtual Router>
```

Comment (kiranchavala) [reply source/plugins/cloudstack-kubernetes-service.rst:166]: @DaanHoogland 
Thanks, the formatting looks good now

Comment (kiranchavala) [reply source/plugins/cloudstack-kubernetes-service.rst]: @DaanHoogland I have committed your suggestion

**Comment by shwstppr** [comment]
@blueorangutan docbuild

File: source/plugins/cloudstack-kubernetes-service.rst [hunk 3]
```diff
@@ -153,6 +153,23 @@ A new network offering named DefaultNetworkOfferingforKubernetesService has been
    - Multi-control nodes, HA cluster can be created for Kubernetes version 1.16 and above only.
    - While creating multi-control nodes, HA cluster over a shared network, an external load-balancer must be manually setup. This load-balancer should have port-forwarding rules for SSH, Kubernetes API server access. Service assumes SSH access to cluster nodes is available from port 2222 to (2222 + cluster node count -1). Similarly, for API access 6443 must be forwarded to control nodes. Over the CloudStack isolated network these rules are automatically provisioned.
 
+
+Examples on how to ssh into the Control - and Worker nodes
```
Comment (shwstppr) [inline source/plugins/cloudstack-kubernetes-service.rst:157]:
```suggestion
Examples of how to ssh into the Control and Worker nodes
```

**Review by DaanHoogland — APPROVED** [review]

**Pull request merged**